tuxemon/cli/commands/trainer_battle.py

```diff
@@ -35,12 +36,7 @@ def invoke(self, ctx: InvokeContext, line: str) -> None:
         if not args:
             print("Missing arguments: <npc_slug>")
             raise ParseError
-        elif len(args) == 1:
-            trainer = args[0]
-            try:
-                action = ctx.session.client.event_engine.execute_action
-                action("create_npc", (trainer, 7, 6))
-                action("start_battle", npc_slug=trainer)
-                action("remove_npc", (trainer,))
-            except:
-                traceback.print_exc()
+        else:
+            ctx.session.client.event_engine.execute_action(
```

Comment:
Maybe leave this command as is, and add a new one for random battle?

Reply:
I'm going to remove and leave the trainer as it was before, at the end they can access through action